src/model.ts

```diff
@@ -292,12 +292,36 @@ export class SpecQueryModel {
         // if the property is a wildcard, return null
         if (isWildcard(encQ[prop])) return null;
 
-        // otherwise, assign the proper to the field def
+        // otherwise, assign the property to the field def
         if (encQ[prop] !== undefined) {
 
           if (!PROPERTY_SUPPORTED_CHANNELS[prop] ||  // all channels support this prop
             PROPERTY_SUPPORTED_CHANNELS[prop][encQ.channel as Channel]) {
             fieldDef[prop] = encQ[prop];
+
+            if (prop === Property.SCALE) {
+              if (isFieldQuery(encQ)) {
+                let field = encQ.field;
+
+                let fieldSchema = this._schema.fieldSchema(field as string);
```

Comment:
Note: We decided it's OK to read directly from `fieldSchema` since we are only assigning scale domain to fieldDef if the domain came from `fieldSchema.ordinalDomain`. A call to Schema's domain() accessor method cannot tell whether that domain came from `ordinalDomain` or `keys(fieldSchema.stats.unique);`